package.json

```diff
@@ -79,7 +84,6 @@
     "lint": "scripts/lint.sh",
     "postversion": "git push upstream --tags",
     "prepublishOnly": "yarn compile",
-    "rollup": "rollup -c rollup.config.js",
```

Comment:
All rollup logic has been moved into the `bundle` gulp task. This command is no longer needed.

Reply:
TODO: add back for legacy pipeline integration (temporary)